pacemaker_metrics.go

```diff
@@ -2,6 +2,8 @@ package main
 
 import (
 	"encoding/xml"
+
+	"github.com/pkg/errors"
```

Comment:
I think we can spare us this external dep

and using `return fmt.Errorf("Error serializing libvirt storage pool: %s", err)`